src/apps/search-result/facet-browser.test.ts

```diff
@@ -0,0 +1,247 @@
+const coverUrlPattern = /^https:\/\/res\.cloudinary\.com\/.*\.(jpg|jpeg|png)$/;
+
+describe("Facet Browser", () => {
+  // I have used it.only because it is best practice to have self contained tests
+  // https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests
+  it.only("Open facet browser and check if all the facets are there", () => {
```

Comment:
Pleas remove `only`.

I appreciate the intention here but you have misunderstood [the purpose of `.only()`](https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests).

> Usually you wouldn't check code using test.only into source control - you would use it for debugging, and remove it once you have fixed the broken tests.

Reply:
Consider naming and the [describe/it pattern](https://prowe214.medium.com/unit-tests-more-readable-describe-it-statements-in-protractor-jasmine-3810b07a6ac5)

Example:

To describe... (the) facet browser:
 - It... renders all facets
 - It ... renders all terms in a facet when clicked
 - It ... updates the search result when a term is selected
 - It ... updates the search result when a term is deselected
 - It ... renders all facets as closed by default
 - It ... renders a facet as open when it has a selected term 
 - It ... renders a facet as closed when it has no more selected terms